src/main/resources/static/js/MainWindow.js

```diff
@@ -302,7 +302,7 @@ function saveConfig() {
 						streams.push(obj);
 					});
 
-	if (stop === true) {
+	if (stop == true) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)